maven-plugin/src/main/java/io/confluent/kafka/schemaregistry/maven/DownloadSchemaRegistryMojo.java

```diff
@@ -198,4 +206,13 @@ private String getExtension(ParsedSchema parsedSchema) {
         return ".txt";
     }
   }
-}
+
+  protected static String encode(String subject) {
+    try {
+      String newSubject = URLEncoder.encode(subject, "UTF-8");
+      return newSubject.replaceAll("%", PERCENT_REPLACEMENT);
+    } catch (Exception e) {
+      return subject;
```

Comment:
Can you add a log that we failed encoding?

Reply:
Done